Add document about addon option to cppcheck (#426)
sider/sider-docs: docs/tools/cplusplus/cppcheck.md

This option allows you to force analyzing code as a given language. The available values are `c` and `c++`.

See also the `--language` option via the command-line help.

### `addon`

This option allows you to set the Addons. For example:

```yaml
linter:
  cppcheck:
    addon:
      - cert
      - misra
```

The available values are `cert`, `misra`, `y2038`, and `threadsafety`.

> See also the [`--addon`](https://github.com/danmar/cppcheck/tree/master/addons).

Reviewer: The title of the specified `README.md` looks "Cppcheck addons", so how about this?

```suggestion
See also [Cppcheck addons](https://github.com/danmar/cppcheck/tree/master/addons).
```

Author: It's better. I will modify it.

Thanks!

> This option allows you to set the Addons. For example:
> 
> ```yaml
> linter:
>   cppcheck:
>     addon:
>       - cert
>       - misra
> ```
> 
> The available values are `cert`, `misra`, `y2038`, and `threadsafety`.
> 
> See also [Cppcheck addons](https://github.com/danmar/cppcheck/tree/master/addons).

Reviewer 2: [suggestion] How about the following simplification?

```suggestion
This option allows you to set the [predefined addons](https://github.com/danmar/cppcheck/tree/master/addons). For example:

```yaml
linter:
  cppcheck:
    addon:
      - cert
      - misra
```

The available values are `cert`, `misra`, `y2038`, and `threadsafety`.
```
